src/compas_fab/backends/vrep/backend_features/vrep_plan_motion.py

```diff
@@ -0,0 +1,201 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+from timeit import default_timer as timer
+
+from compas_fab.backends.backend_feature_interfaces import PlanMotion
+from compas_fab.backends.vrep.helpers import config_from_vrep
+from compas_fab.backends.vrep.helpers import config_to_vrep
+from compas_fab.backends.vrep.helpers import floats_to_vrep
+from compas_fab.backends.vrep.helpers import frame_to_vrep_pose
+from compas_fab.backends.vrep.helpers import VrepError
+
+__all__ = [
+    'VrepPlanMotion',
+]
+
+
+class VrepPlanMotion(PlanMotion):
+    """Callable to find a path plan to move the selected robot from its current position to the `goal_constraints`.
+    """
+    SUPPORTED_PLANNERS = ('bitrrt', 'bkpiece1', 'est', 'kpiece1',
+                          'lazyprmstar', 'lbkpiece1', 'lbtrrt', 'pdst',
+                          'prm', 'prrt', 'rrt', 'rrtconnect', 'rrtstar',
+                          'sbl', 'stride', 'trrt')
+
+    def __init__(self, client):
+        self.client = client
+
+    def plan_motion(self, goal_constraints, start_configuration=None, group=None, options={}):
+        """Finds a path plan to move the selected robot from its current position to the `goal_constraints`.
+
+        Args:
+            goal_constraints (:class:`Frame`): Target or goal frame.
```

Comment:
If would be ideal if this could actually be a list of `Constraints` and they check for positional/orientation values to rebuild the frame (and it would allow to merge this and `plan_motion_to_configs` into one method. However, if you decide to skip and leave for the future, it would be totally understandable.

Reply:
I believe this is beyond the scope of this PR, and should be made into a separate issue to be addressed in the future.

Reply:
agree 👍